### PR TITLE
Feature navbarstyle

### PR DESCRIPTION
**Pull request by pnguye17**

### CODE REVIEW

**Review by carolynrehm — APPROVED** [review]
@pnguye17 this looks good, just going to wait until your documentation branch gets merged in as this branch has some of those changes

**Comment by carolynrehm** [comment]
@pnguye17 resolve merge conflict and this will be good to go

**Review by AdonisCabreja — APPROVED** [review]

**Review by carolynrehm — APPROVED** [review]

**Pull request merged**
